src/create-label-by-rate-id/types/private-request.ts

```diff
@@ -0,0 +1,24 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type CreateLabelFromRateRequestBody = PurchaseLabelWithoutShipment;
+export type ValidateAddress =
+  | "no_validation"
+  | "validate_only"
+  | "validate_and_clean";
+export type LabelLayout = "4x6" | "letter";
+export type LabelFormat = "pdf" | "png" | "zpl";
+export type LabelDownloadType = "url" | "inline";
+export type DisplayScheme = "label" | "qr_code";
+
+export interface PurchaseLabelWithoutShipment {
+  validate_address?: ValidateAddress;
+  label_layout?: LabelLayout & string;
+  label_format?: LabelFormat & string;
+  label_download_type?: LabelDownloadType;
+  display_scheme?: DisplayScheme & string;
```

Comment:
oh wow, hmmmm .... type Foo = ('bar' | 'baz') & string is the same as `type Foo = string` (since string is the _supertype_, it eats the other types)